src/app/resources/resources.component.html

```diff
@@ -9,7 +9,7 @@
     </sb-library-filters>
   </div>
   <div class="scrollable-container">
-    <ion-refresher (ionRefresh)="swipeDownToRefresh($event)"></ion-refresher>
+    <ion-refresher #libraryRefresher slot="fixed" pullFactor="0.5" pullMin="100" pullMax="200" (ionRefresh)="swipeDownToRefresh($event)"></ion-refresher>
```

Comment:
Keep the same configuration the same as we have in profile and course page or change the profile and course to have the same config as the library.